<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Sharder_Button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Sharder_Export</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_Sharder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
   </widget>
   <widget class="QListView" name="listView_Cache">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Cache_Button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Cache_Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cache_Sharder_Button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cache_Sharder_Import</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Import_view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">rgb(150, 222, 226)</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>                                                   Directory Path  of  scene_lit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Sharder_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>                                                           Sharder  Directory Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Cache_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>                                                            Cache  Directory Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Check_one">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Check_two">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>691</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>681</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
